_data/banking.yml

```diff
@@ -220,7 +220,6 @@ websites:
       tfa: Yes
       sms: Yes
       phone: Yes
-      doc: https://www.therightbank.com/files/capital-pacific-bank-online-services-instructions.pdf
```

Comment:
I think this is a mistake because now this file is at https://www.therightbank.com/sites/www.therightbank.com/files/files/online-services-instructions.pdf